perun-wui-core/src/main/java/cz/metacentrum/perun/wui/json/managers/AttributesManager.java

```diff
@@ -85,4 +85,35 @@ public static Request getLogins(int userId, JsonEvents events) {
 
 	}
 
+	/**
+	 * Returns an Attribute by its name. Returns only non-empty attributes.
```

Comment:
Methods retrieving attributes by name (urn) return also empty values !

Reply:
This text is taken from the RPC documentation. Does it mean that the RPC documentation is incorrect?

Reply:
As far as I know, if you retrieve attribute by name, it should return value even if empty. Please check implementation in core and update javadoc if necessary.

Reply:
I have checked the implementation and it returns also empty values. I will create pull request with the javadoc fix.